DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -1,10 +1,30 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
 
 public class ComputerGeekGame
 {
+    private string _scoreBoardfile = "DeveloperGameLeaderBoard.txt";
```

Comment:
Should the file name be: "ComputerGeekLeaderBoard"?

Also because this goes in the same folder as the .exe, I think it would be sufficient to just say "leaderboard.txt", unless you plan on having multiple different types of leaderboards...

Reply:
Also your variable name would imply that it is a file, but it is only string, so I would say "filename" instead of just "file"

Reply:
Fixed.